docs/en/latest/plugins/response-rewrite.md

```diff
@@ -118,6 +118,61 @@ X-Server-balancer_addr: 127.0.0.1:80
 {"code":"ok","message":"new json body"}
 ```
 
+Replace the string X-Amzn-Trace-Id of body to X-Amzn-Trace-Id-Replace.
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+  "plugins": {
+    "response-rewrite": {
+            "headers": {
```

Comment:
Could you please format the JSON with 2 spaces only? You can use tools like https://www.json.cn/

Reply:
sure